internal/configs/version1/nginx-plus.tmpl

```diff
@@ -346,3 +346,9 @@ stream {
 
     include /etc/nginx/stream-conf.d/*.conf;
 }
+
+{{- if and (.NginxVersion.IsPlus) (.NginxVersion.PlusGreaterThanOrEqualTo "nginx-plus-r31") }}
```

Comment:
do we need `(.NginxVersion.IsPlus)` here as string in next condition is only possible on plus versions?